contribute/guidelines.md

```diff
@@ -56,7 +53,7 @@ Due to a `pip` limitation, it is required to increment the major version number
 Pull requests should generally be **reviewed** by someone else than their authors.
 
 This is mandatory for:
-- Any Pull Request on **`openfisca-core`**
```

Comment:
Ok, I guess this is replaced by the CONTRIBUTING in core. This is not entirely satisfying, as when a contributor reads this first, he may not know that there are other guidelines in the repository. 

Anyways this is not blocking for me, contributors will realize they need a review when github will tell them they can't merge without it 😈